### PR TITLE
April ur test improvements

### PR DESCRIPTION
**Pull request by thomasiles**
This branch contains the changes to the prototype for the first round of testing with users. 
See ticket https://trello.com/c/37LaeJvS/201-ensure-prototype-is-ready-for-initial-user-testing

### CODE REVIEW

File: app/views/form-designer/form-home.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  Form Home
+{% endblock %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">Welcome to GOV.UK Forms</h1>
+        {{ govukButton({
+        text: "Create a From",
```
Comment (DavidBiddle) [inline app/views/form-designer/form-home.html:13]: Typo:
```suggestion
        text: "Create a Form",
```

File: app/views/publisher/start.html [hunk 2]
```diff
@@ -14,7 +14,8 @@
         <div class="govuk-grid-column-full">
             <h1 class="govuk-heading-xl">Your forms</h1>
             {{ govukButton({
-                text: "Create a new form"
+                text: "Create a new form",
+                href: '../form-designer/form-index'
```
Comment (DavidBiddle) [inline app/views/publisher/start.html:18]: Think this should point to either `../form-designer/form-create-a-form.html` or `../form-designer/form-home.html` (this was one of my changes I think so my fault)

**Comment by deberny** [comment]
Screenshot 2022-04-28 at 11 38 48

Fantastic changes!

I've noticed the 'Update preview' button doesn't seem to be working? as in it's not updating the preview in the iframe.

I've also tweaked the styling a bit: I've changed the button to look like actual button, but with secondary styles and gave the link more padding. [ will suggest the changes in the review ]

I think we should disable the 'continue button' within the iframe, so it doesn't get clicked by accident. I've tried to add the class and disabled attribute to the button, but I'm breaking it. Ooops.

File: app/views/form-designer/edit-page.html [hunk 3]
```diff
@@ -334,19 +315,19 @@ <h1 class="govuk-heading-l">
 
       <div class="govuk-grid-column-one-third" style="height: 1462px;">
         <div class="preview-pane-wrapper">
+          {{ govukButton({
+          text: "Update preview",
+          classes: "update-button govuk-!-margin-bottom-0 govuk-!-padding-2",
```
Comment (deberny) [inline app/views/form-designer/edit-page.html:320]: We could remove 'update-button' class and add 'govuk-button--secondary' instead

File: app/views/form-designer/form-home.html (hunk 1 above)
Comment (thomasiles) [reply app/views/form-designer/form-home.html:13]: good catch! I think this is going to haunt me throughout the project...

**Review by DavidBiddle — APPROVED** [review]

**Comment by DavidBiddle** [comment]
Looks great! 👏

**Pull request merged**